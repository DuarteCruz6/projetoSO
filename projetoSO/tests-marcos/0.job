SHOW
WRITE [(h1,32)(p4,27)(i8,55)(q9,27)]
READ [p4]
WRITE [(k9,81)(b6,38)]
WRITE [(q8,6)(y3,33)(p1,92)(t8,49)(v5,84)]
READ [q9]
READ [k9]
DELETE [y3,q8,q9]
WRITE [(m4,75)(e2,37)(h9,63)(e8,10)(w5,33)]
WRITE [(x4,20)(a1,71)(h7,41)]
WRITE [(j0,100)(s9,95)]
BACKUP
READ [i8]
READ [t8]
DELETE [a1,k9,i8]
WRITE [(h8,55)(v4,73)(q6,52)(f2,15)]
READ [e2]
BACKUP
WRITE [(t0,90)(a1,37)(y3,89)]
DELETE [h7,p1,h9,e2,b6]
BACKUP
WRITE [(k3,50)(e2,75)(g3,54)(g2,40)(v2,26)]
WRITE [(p9,35)(r0,81)(v8,71)(h4,17)(o5,75)]
DELETE [w5]
DELETE [r0,x4,k3]
WRITE [(f4,23)]
READ [h1]
WRITE [(l1,100)(q9,91)(d7,33)(i3,64)]
READ [p4]
WRITE [(s4,50)]
WRITE [(m5,72)]
WRITE [(x2,41)]
DELETE [g3,s9,q9,j0,p9]
WRITE [(o3,13)(p6,21)(h7,84)(t5,70)(r5,81)]
DELETE [o3]
READ [t0]
WRITE [(o1,55)(c6,58)(p9,42)(g6,30)(w1,65)]
WRITE [(q9,41)(g7,22)(l3,31)(q1,9)]
DELETE [r5,f4,o5]
WRITE [(d6,78)]
WRITE [(e6,1)]
READ [t8]
WAIT 3000
WRITE [(l5,66)]
WRITE [(k8,64)(z3,60)]
DELETE [f2,m5,s4,h7]
WRITE [(c0,4)(c8,33)]
WRITE [(k1,46)(s1,23)]
READ [v4]
WRITE [(r7,74)]
WRITE [(v3,32)(u4,99)(f3,20)(u4,59)(a4,77)]
DELETE [f3,s1,c6,k8]
WRITE [(q2,7)(p7,67)]
DELETE [e6,a1,p6]
READ [h8]
DELETE [a4,t0]
WRITE [(q4,24)]
READ [p4]
WAIT 3000
DELETE [h1,q4,w1]
WRITE [(i4,88)(f9,25)(b2,57)(j4,21)(i2,47)]
SHOW
WAIT 1000
WRITE [(g0,45)(b3,96)]
WRITE [(r3,0)(c5,54)(w3,3)(n3,92)(l2,40)]
WRITE [(g1,13)(z0,84)(o5,72)(q7,94)(r2,6)]
WAIT 3000
WAIT 3000
WRITE [(k7,74)(h6,47)(q3,80)(r6,1)(b5,41)]
SHOW
WRITE [(u6,73)]
WAIT 1000
BACKUP
WRITE [(p0,34)(w0,12)(u1,84)(o0,67)(d4,45)]
WRITE [(v9,54)(i7,5)]
WRITE [(o9,81)(n5,13)]
WRITE [(a3,85)(k6,43)]
WRITE [(f8,90)(e4,69)(w9,75)(y7,39)]
DELETE [c5,r6,p9,i2,i3]
WRITE [(u8,57)(k4,32)(i5,81)(f5,89)(c7,22)]
WRITE [(b6,35)(s8,77)]
READ [v8]
WRITE [(b4,5)(o3,0)]
DELETE [v9,p4,i7,k6,v2]
WRITE [(k2,50)(m5,70)]
DELETE [v5,g2,m5,o1]
WRITE [(i7,67)(u9,24)(m5,38)]
WRITE [(p4,6)]
