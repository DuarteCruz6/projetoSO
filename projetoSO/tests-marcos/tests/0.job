WRITE [(h3,79)(l5,21)(s2,4)(b9,6)(k9,75)]
READ [s2]
READ [b9]
WRITE [(o3,6)(n3,32)(d7,33)]
READ [k9]
WRITE [(j6,85)(w9,52)(w3,55)]
DELETE [k9,d7]
WRITE [(f0,72)(z9,92)]
DELETE [z9,b9,j6,h3]
WAIT 2000
DELETE [w3,n3,w9,f0]
WRITE [(t8,55)(q6,83)(l1,53)(d1,66)]
WRITE [(t3,68)(n6,28)(d6,45)(d5,29)(k0,45)]
READ [n6]
SHOW
WRITE [(c1,28)]
DELETE [t3,k0,q6,d1]
READ [l1]
WRITE [(n7,42)(t1,51)(y2,50)]
DELETE [s2,l1]
READ [t8]
WAIT 3000
READ [n6]
READ [d5]
WRITE [(s5,56)(a3,60)(v4,40)]
WRITE [(q1,68)(f4,51)(d9,8)]
WRITE [(w5,72)(b6,71)(m5,53)(j8,6)]
DELETE [a3,f4,w5,y2]
READ [d9]
READ [o3]
WRITE [(p7,66)(h9,96)]
WRITE [(u7,62)]
