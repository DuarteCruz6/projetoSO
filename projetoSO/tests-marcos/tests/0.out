(j1, 42)
(m4, 99)
(o7, 27)
(q8, 61)
(q4, 30)
(r4, 11)
(u7, 47)
(v6, 52)
(z9, 13)
[(g3,67)]
[(u9,KVSMISSING)]
(a4, 14)
(a8, 57)
(a7, 74)
(b1, 92)
(c9, 60)
(c7, 41)
(c0, 31)
(d4, 90)
(d2, 28)
(e8, 86)
(e0, 42)
(f2, 13)
(g5, 42)
(h8, 77)
(h9, 15)
(j2, 85)
(k7, 72)
(k0, 13)
(k8, 75)
(k9, 10)
(l1, 36)
(n3, 83)
(n5, 27)
(o4, 31)
(o1, 18)
(o0, 7)
(p7, 9)
(p6, 18)
(p1, 11)
(p8, 95)
(p5, 23)
(p3, 28)
(r5, 31)
(r9, 5)
(s0, 23)
(s2, 49)
(u0, 97)
(v9, 65)
(v2, 70)
(x4, 69)
(x5, 93)
(x6, 8)
(y0, 97)
(y5, 61)
(z8, 0)
(z4, 38)
[(p3,28)]
[(s9,KVSMISSING)]
[(j3,10)]
[(d9,92)]
[(p3,28)]
[(y8,31)]
[(g1,88)]
[(u4,57)]
[(y8,31)]
[(b7,18)]
[(u5,17)]
[(y6,41)]
[(d1,84)]
(a9, 15)
(a5, 47)
(a0, 12)
(a4, 14)
(a8, 57)
(a7, 11)
(b2, 40)
(b9, 17)
(b7, 18)
(b1, 2)
(c4, 81)
(c3, 1)
(c9, 60)
(d6, 58)
(d9, 92)
(d1, 84)
(d4, 22)
(e2, 41)
(e4, 66)
(e7, 75)
(e3, 77)
(e8, 86)
(f6, 59)
(f0, 48)
(f2, 45)
(g8, 41)
(g4, 96)
(g2, 24)
(g1, 88)
(g9, 17)
(g0, 73)
(g6, 63)
(g5, 42)
(h3, 16)
(h7, 64)
(h9, 15)
(i2, 56)
(i7, 85)
(i0, 100)
(i9, 2)
(j7, 26)
(j9, 30)
(j3, 10)
(j6, 21)
(j2, 75)
(k6, 42)
(k5, 7)
(k3, 30)
(k4, 43)
(k2, 27)
(k1, 95)
(k7, 72)
(k8, 75)
(l9, 12)
(l4, 12)
(l8, 33)
(l1, 36)
(m6, 73)
(m5, 88)
(m2, 90)
(m1, 15)
(n2, 97)
(n7, 97)
(n5, 67)
(o6, 20)
(o3, 56)
(o0, 7)
(p2, 21)
(p7, 9)
(p6, 18)
(p8, 95)
(p5, 23)
(p3, 28)
(q4, 83)
(q7, 35)
(r3, 86)
(r2, 91)
(r4, 39)
(r6, 90)
(r0, 82)
(r5, 31)
(r9, 5)
(s1, 57)
(s4, 97)
(s7, 85)
(s5, 100)
(s0, 9)
(s2, 49)
(t9, 24)
(t4, 56)
(t1, 0)
(u8, 54)
(u2, 7)
(u4, 57)
(u5, 17)
(u0, 97)
(v0, 21)
(v1, 38)
(v3, 68)
(v9, 10)
(v2, 70)
(w9, 80)
(w1, 86)
(w4, 85)
(x3, 8)
(x2, 79)
(x9, 38)
(x4, 69)
(x5, 93)
(y1, 42)
(y3, 90)
(y2, 41)
(y8, 31)
(y7, 76)
(y6, 41)
(y0, 97)
(z1, 18)
(z5, 88)
(z2, 42)
(z8, 0)
[(q8,KVSMISSING)]
[(k6,KVSMISSING)]
[(e2,41)]
